screens/ClerkLoginScreen.js

```diff
@@ -67,33 +89,29 @@ export default class ClerkLoginScreen extends React.Component {
         <Container>
           <Picker
             mode="dropdown"
-            onValueChange={store => this.setState({ storeId: store })}
+            onValueChange={storeId => this.setState({ storeId })}
             selectedValue={this.state.storeId}>
-            {this.state.stores.map((item, index) => {
-              return (
-                <Picker.Item
-                  label={item['name']}
-                  value={item['id']}
-                  key={index}
-                />
-              );
+            {this.state.stores.map(store => {
+              return <Picker.Item label={store.storeName} value={store.id} key={store.id} />;
             })}
           </Picker>
           <TextInput
             placeholder="Password"
             keyboardType="number-pad"
             maxLength={4}
-            secureTextEntry={true}
+            secureTextEntry
             onChangeText={text => this.setState({ password: text })}
             value={this.state.password}
           />
-          <SubmitButton
-            color="#008550"
-            title="Log In"
-            onPress={() => this.handleSubmit()}
-          />
+          <SubmitButton color="#008550" title="Log In" onPress={() => this.handleSubmit()} />
```

Comment:
Should we use the `primaryGreen` color from Colors.js for this?

Reply:
Eventually yeah but we can worry about this in the next styling PR.